### PR TITLE
Add a /reload command that quickly reloads the current demo

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Rather simple addition that allows you to quickly re-read the currently parsed .demo file. Extremely helpful when modifying the .demo file and wanting to quickly see the changes without having to go through file dialogs.

### CODE REVIEW

File: src/demoserver.cpp [hunk 1]
```diff
@@ -220,14 +220,20 @@ void DemoServer::handle_packet(AOPacket packet)
               client_sock->write(packet.toUtf8());
           }
         }
+        else if (contents[1].startsWith("/reload"))
+        {
+            load_demo(p_path);
+            QString packet = "CT#DEMO#" + tr("Current demo file reloaded. Send /play or > in OOC to begin playback.") + "#1#%";
+            client_sock->write(packet.toUtf8());
```
Comment (oldmud0) [inline src/demoserver.cpp:227]: You may want to reset the playback timer. There are also stateful components like the countdown timer you might need to reset/hide.

Comment (Crystalwarrior) [reply src/demoserver.cpp:227]: Ohhh good catch

File: src/demoserver.cpp [hunk 2]
```diff
@@ -162,6 +162,23 @@ void DemoServer::handle_packet(AOPacket packet)
...
+          client_sock->write("TI#2#1#0#%");
+          client_sock->write("TI#2#3#0#%");
+          client_sock->write("TI#3#1#0#%");
+          client_sock->write("TI#3#3#0#%");
+          client_sock->write("TI#4#1#0#%");
+          client_sock->write("TI#4#3#0#%");
+          // Set the BG to default (also breaks up the message queue)
+          client_sock->write("BN#default#wit#%");
+          // Stop the wait packet timer
+          timer->stop();
```
Comment (oldmud0) [inline src/demoserver.cpp:181]: Please move this to its own method thanks

Also if you want, we can totally modify the TI packet specs to add a command for resetting all timers.

**Review by oldmud0 — CHANGES_REQUESTED** [review]

File: src/demoserver.cpp (hunk 2 above)
Comment (Crystalwarrior) [reply src/demoserver.cpp:181]: can u do it plz

Comment (in1tiate) [reply src/demoserver.cpp]: ur welcome :3

**Review by oldmud0 — APPROVED** [review]

**Pull request merged**